editor/xml_feature.cpp

```diff
@@ -349,14 +349,15 @@ string XMLFeature::GetTagValue(string const & key) const
   return tag.attribute("v").value();
 }
 
-void XMLFeature::SetTagValue(string const & key, string const & value)
+void XMLFeature::SetTagValue(string const & key, string value)
 {
+  strings::Trim(value);
   auto tag = FindTag(m_document, key);
   if (!tag)
   {
     tag = GetRootNode().append_child("tag");
-    tag.append_attribute("k").set_value(key.c_str());
-    tag.append_attribute("v").set_value(value.c_str());
+    tag.append_attribute("k").set_value(key.data());
```

Comment:
Because in this file almost all methods were using `.data()` except three, even in similar contexts, like here.